workloads/router-perf-v2/env.sh

```diff
@@ -17,6 +17,9 @@ KUBE_BURNER_IMAGE=quay.io/cloud-bulldozer/kube-burner:latest
 TERMINATIONS=${TERMINATIONS:-"http edge passthrough reencrypt mix"}
 INFRA_TEMPLATE=http-perf.yml.tmpl
 INFRA_CONFIG=http-perf.yml
+export DEPLOYMENT_REPLICAS=${DEPLOYMENT_REPLICAS:-10}
+#export SMALL_SCALE_ROUTES=10
```

Comment:
Now I get why you put this hear, but it's kinda cryptic and I just glanced over it and thought nothing of it. Consider putting a comment such as:
`# Recommend to use a smaller number of routes if running with INFRA_TEMPLATE=http-perf-depl.yml.tmpl`

Reply:
Would the README be a better place for this?

Reply:
oh snap there's a README!? I'll let you decide on that, I'm good as long as it's somewhere I suppose. It could go in the SMALL_SCALE_ROUTES and LARGE_SCALE_ROUTES rows. 

Now that you mention there is a README, should you add `DEPLOYMENT_REPLICAS` as a new ENV variable and a explanation?

Reply:
Indeed, I have added `DEPLOYMENT_REPLICAS` to the README (second commit to this PR).

Reply:
Ah sorry I see that now. Looks good.